core/src/main/java/org/elasticsearch/discovery/zen/NodeJoinController.java

```diff
@@ -256,7 +256,7 @@ public synchronized boolean isEnoughPendingJoins(int pendingMasterJoins) {
 
         private Map<DiscoveryNode, ClusterStateTaskListener> getPendingAsTasks() {
             Map<DiscoveryNode, ClusterStateTaskListener> tasks = new HashMap<>();
-            joinRequestAccumulator.entrySet().stream().forEach(e -> tasks.put(e.getKey(), new JoinTaskListener(e.getValue(), logger)));
+            joinRequestAccumulator.entrySet().forEach(e -> tasks.put(e.getKey(), new JoinTaskListener(e.getValue(), logger)));
```

Comment:
None of these changes are necessary, but if we are going to get in these places and make changes, let's do it the preferred way which is with method references (where possible).